src/pages/public_registration/log/LogEntry.tsx

```diff
@@ -0,0 +1,92 @@
+import AccountBalanceIcon from '@mui/icons-material/AccountBalance';
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import DeleteOutlinedIcon from '@mui/icons-material/DeleteOutlined';
+import InsertDriveFileOutlinedIcon from '@mui/icons-material/InsertDriveFileOutlined';
+import LocalOfferOutlinedIcon from '@mui/icons-material/LocalOfferOutlined';
+import UnpublishedOutlinedIcon from '@mui/icons-material/UnpublishedOutlined';
+import { Avatar, Box, Divider, styled, SvgIconProps, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { LogEntryType } from '../../../api/registrationApi';
+import { LogDateItem } from '../../../components/Log/LogDateItem';
+import { FileType } from '../../../types/associatedArtifact.types';
+import { getInitials } from '../../../utils/general-helpers';
+import { getFullName } from '../../../utils/user-helpers';
+
+const logIconProps: SvgIconProps = { color: 'primary', fontSize: 'small' };
+
+const StyledLogRow = styled(Box)({
+  display: 'flex',
+  gap: '0.5rem',
+  alignItems: 'center',
+});
+
+interface LogEntryProps {
+  logEntry: LogEntryType;
+}
+
+export const LogEntry = ({ logEntry }: LogEntryProps) => {
+  const { t } = useTranslation();
+
+  const fullName = getFullName(logEntry.performedBy.givenName, logEntry.performedBy.familyName);
+  const initials = getInitials(fullName);
+
+  return (
+    <Box
+      sx={{ display: 'flex', flexDirection: 'column', p: '0.5rem', bgcolor: 'publishingRequest.light', gap: '0.5rem' }}>
+      <StyledLogRow>
+        <LogHeaderIcon topic={logEntry.topic} />
+        <Typography sx={{ fontWeight: 900, fontSize: '1rem' }}>
+          {logEntry.topic === 'FileApproved' && logEntry.fileType === FileType.OpenFile
+            ? t('log.file_published')
+            : logEntry.topic === 'FileApproved' && logEntry.fileType === FileType.InternalFile
+              ? t('log.file_archived')
+              : t(`log.entry_topic.${logEntry.topic}`, { defaultValue: logEntry.topic })}
+        </Typography>
+      </StyledLogRow>
+      <LogDateItem date={new Date(logEntry.timestamp)} />
+
+      <StyledLogRow>
+        <Avatar sx={{ height: '1.5rem', width: '1.5rem', fontSize: '0.7rem', bgcolor: 'primary.main' }}>
+          {initials}
+        </Avatar>
+        <Typography>{fullName}</Typography>
+
+        <AccountBalanceIcon {...logIconProps} />
+        <Typography>{logEntry.performedBy.onBehalfOf.displayName}</Typography>
```

Comment:
Denne viser nå displayName for kunden i NVA som aldri vil endres. Det burde vært det faktiske navnet i cristin for logg-øyeblikket

Reply:
Vurdert som OK så lenge det er i beta